munit/non-jvm/src/main/scala/munit/internal/junitinterface/Settings.scala

```diff
@@ -1,11 +1,11 @@
 package munit.internal.junitinterface
 
 trait Settings {
-  def trimStackTraces(): Boolean
```

Comment:
Can we add the parentheses back here? These interfaces are meant to be mirrors of their equivalent Java interfaces https://github.com/scalameta/munit/blob/7ea3075ae51ca73c7fa84373aa9dd82cf9a6ccd7/junit-interface/src/main/java/munit/internal/junitinterface/Tag.java

I prefer to keep the two synchronized as much as possible, these are internal APIs so it shouldn't matter much whichever we use.

Reply:
I can add them back, but that makes stuff in some other places more complicated. There are some classes that attempt to use `val` to override those `def`s (which doesn't work anymore in Dotty, so they have to be renamed). Those constructors are subsequently called with named parameters, so their call sites have to be renamed too.

I'm happy to go ahead either way, though.

Reply:
I see. That makes sense, I'm glad Scala 3 is more strict about the differences between `method()` and `method` without parentheses. It's still quite a large breaking change, I always thought it was elegant that `val` could override parameterless defs. 

Anyways, I'm fine dropping the parentheses in this case 👍